src/util/ieee_float.h

```diff
@@ -14,10 +14,14 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "mp_arith.h"
 #include "format_spec.h"
+#include "irep.h"
+#include "cprover_prefix.h"
 
 class constant_exprt;
 class floatbv_typet;
 
+const irep_idt ID_cprover_rounding_mode = CPROVER_PREFIX "rounding_mode";
```

Comment:
I think this should be ok, but it does come with the caveat that the order of static initialisation is not defined and therefore the number of the entry that it receives in the string table may vary.

Reply:
Not only that, but statements like these often end up being initialised *before* the string table is created at all, leading to a segfault during static init. Suggest instead either a static function local (executed the first time the function runs) or directly defining this in irep_ids.def.

Reply:
Ah, good point on the table initialisation, hadn't thought about that. I'd originally wanted to put it into irep_ids.def, but there's currently no way to make this work with the CPROVER prefix. I think it could be added, but I'll probably just have this a local for now.